### PR TITLE
remove f1 support

### PR DESCRIPTION
**Pull request by juliangruber**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,9 +73,6 @@ the configuration options described in
   will receive rewards. The value must be a mainnet address starting with
   `f410`, `0x`.
 
-  Addresses starting with `f1` are deprecated. We allow them for now but will
-  start rejecting them in an upcoming version.
-
```
Comment (bajtos) [inline README.md:78]: Maybe we can keep the mention of f1 addresses and explain why we don't support them.

> Addresses starting with `f1` are not allowed. Rewards for Station operators are administered by a FEVM smart contract. It is currently technically complex to make payments to `f1` addresses.

File: commands/station.js [hunk 2]
```diff
@@ -23,10 +23,7 @@ const panic = msg => {
 
 export const station = async ({ json, experimental }) => {
   if (!FIL_WALLET_ADDRESS) panic('FIL_WALLET_ADDRESS required')
-  if (FIL_WALLET_ADDRESS.startsWith('f1')) {
-    console.error('Warning: f1... addresses are deprecated and will not receive any rewards.')
-    console.error('Please use an address starting with f410 or 0x')
```
Comment (bajtos) [inline commands/station.js:28]: Personally, I would keep the special error message to nicely explain to the current users that they have to upgrade their config.

```
panic('Error: f1... addresses are no longer supported. Please use an address starting with f410 or 0x')
```

**Review by bajtos — APPROVED** [review]
Thank you for taking care of this change!

I have a couple of minor/subjective suggestions, feel free to ignore them.

File: README.md (hunk 1 above)
Comment (juliangruber) [reply README.md]: good idea!

File: commands/station.js (hunk 2 above)
Comment (juliangruber) [reply commands/station.js:28]: good idea!

**Comment by juliangruber** [comment]
https://github.com/filecoin-station/core/releases/tag/v15.0.0

**Pull request merged**